packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -190,10 +190,7 @@ export default async function processDeployment({
       }
     }
 
-    if (
-      event.type === 'build-state-changed' &&
-      event.payload.readyState === 'BUILDING'
-    ) {
+    if (event.type === 'building' && event.payload.readyState === 'BUILDING') {
```

Comment:
Checking both type and payload seems redundant, can we do the following?
```suggestion
    if (event.type === 'building' && event.payload.readyState === 'BUILDING') {
```